doc/decisions/highlevel_help_message.md

```diff
@@ -0,0 +1,54 @@
+# High-level API Help Message
+
+This decision _does not_ assume code-generation is used. For the case of code-generation see the [Notes](#notes) section.
+
+## Problem
+
+We want to allow to print the help message no matter what errors happened in `kdbOpen` or `kdbGet`.
+
+## Constraints
+
+- `elektraOpen` should not return a broken `Elektra` instance.
+- The help message can only be printed, if `elektraOpen` returns an `Elektra` instance and no `ElektraError`.
+
+## Assumptions
+
+- We assume that the application in question was correctly installed.
+- We assume `gopts` was mounted.
```

Comment:
Is this already default?

Reply:
`gopts` is not mounted by default no. However, the file generated by the `highlevel` template, include a `kdbEnsure` contract, that mounts it, if it is missing.

Reply:
Please copy this sentence into the decision.